tcn_hpl/data/add_gt_to_kwcoco.py

```diff
@@ -0,0 +1,125 @@
+import kwcoco
+from angel_system.data.medical.data_paths import TASK_TO_NAME
+from angel_system.data.medical.data_paths import LAB_TASK_TO_NAME
+import yaml
+import argparse
+import tcn_hpl.utils.utils as utils
+import ubelt as ub
+
+def text_to_labels(text_file: str, num_frames: int, task: str, mapping: dict):
+    
+    # set background to everything first
+    activity_gt_list = [0 for x in range(num_frames)]
```

Comment:
It doesn't seem like we are dynamically changing the size of this much, so it may be better to initialize this as a numpy array:
```suggestion
    activity_gt_list = np.zeros(num_frames, dtype=int)
```